### PR TITLE
Fix nats jetstream conformance tests

### PR DESCRIPTION
**Pull request by Jarema**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by ItalyPaleAle** [comment]
Thanks for this PR!

Could you please help us understand what these changes are for? What's the problem with the current conformance tests that you're fixing?

**Comment by berndverst** [comment]
I'm aware of these changes @ItalyPaleAle @yaron2. I'll handle the review. I have an ongoing chat with the author.

File: tests/config/pubsub/jetstream/pubsub.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ spec:
     value: config-test
   - name: flowControl
     value: true
+  - name: hearbeat
```
Comment (berndverst) [inline tests/config/pubsub/jetstream/pubsub.yml:15]: Should this be `heartbeat` instead?

```suggestion
  - name: heartbeat
```

Comment (berndverst) [reply tests/config/pubsub/jetstream/pubsub.yml:15]: Never mind it really is `hearbeat` - odd :D

**Review by berndverst — DISMISSED** [review]

**Review by berndverst — APPROVED** [review]

**Pull request merged**